mep/common/models.py

```diff
@@ -90,16 +90,32 @@ class Meta:
     def dates(self):
         '''Date or date range as a string for display'''
 
-        # if no dates are set, return an empty string
-        if not self.start_year and not self.end_year:
+        if not self.start_year and not self.end_year: # no dates are set
             return ''
 
-        # if start and end year are the same just return one year
-        if self.start_year == self.end_year:
-            return self.start_year
+        if not self.end_year: # only start year
```

Comment:
I think this logic could be refactored a bit to clean it up.

As a suggestion, a helper method that strings the year since it's a repeated pattern (could be static or class, wrote below as a simple method):
```
def _stringify_year(self, value):
     # logic to return 'xxx BCE' or 'xxx CE'
```
That would avoid having to repeat yourself so much below. I'm not happy with my name because it's a bit long, but something like that would make the code below more readable, imo.